### DISCUSSION TITLE
KodaDot discord reorganization and spam fight

### DISCUSSION THREAD

**Discussion by damskyftw** [kodadot/nft-gallery#2020] [General]
## Hello fellow KodaDotters 

Since our discord server is being targeted by spammers and bots, due to activity and so on, we've decided to take action. This discussion is regarding the KodaDot discord server, which most of you is part of, so **feel free to share your opinion**. Our goal is to make the server as comfortable as possible, with zero spammers and relevant channels/categories according to your desire.

### What we are meaning to do is:

**In first step, without any prolonging:**

- [x]   Verification channel, where if you don't react to a message, the server will stay closed for you, with only k-overview and arrivals visible

**In second step, after some community talk here or on discord:**

- [ ]  Add for each category a role, so you only choose specific role which belongs to specific category. That way, you can make KodaDot discord custom and according to your needs and interests.

 For example, since i'm not a dev, i wouldn't need to see the dev-core category, but i like to peek in from time to time, so i can control this by clicking on some emoji which allows/disallows viewing certain categories.

For now, we'll be making step first, which is "verification" channel, where all users will have to react to a bot and regain access to the server. This is mainly to avoid spammers until we implement Portculis NFT gating, which will most certainly avoid all of them.

### What do you think? 

![asdf](https://user-images.githubusercontent.com/92786519/150638653-be45d534-18b7-413d-9742-2845f4c10732.png)

**Comment by yangwao** [comment]
Current state of channels snapshot

| ![image](https://user-images.githubusercontent.com/5887929/150687208-2d568211-dfc4-49bc-bb3d-37b913d4f2e3.png) | ![image](https://user-images.githubusercontent.com/5887929/150687218-22fd420f-364d-4cc7-9c30-31bd56341c9f.png) | 

| 
![image](https://user-images.githubusercontent.com/5887929/150687285-9670e510-95e0-4b97-a4aa-3bd21b079652.png) |  ![image](https://user-images.githubusercontent.com/5887929/150687299-e745f736-4a90-440a-a440-53910afa4580.png) |

**Comment by yangwao** [comment]
Intention would be
- archive low-traffic channels
- Culture and K-coordination have channels which could emerge separate category
- Extract high-quality category where could be only OGs like with over 10-30 mints or collectors who done at least 30 buys 
- [x] metaprime archived
- [x] create contest category
- [x] hypermedia hidden

### Culture 
- limit it 50-70%
- 5-7 channels?

### K-coordination 
- 4-7 channels?

**Comment by damskyftw** [comment]
Yeah, also a good point:

- Rework a little announcement channel, since we're establishing connection with some friendly discords, they can just simply follow the channel and once we click "publish" it will post on their discord-specific channel they choose and we can keep the community and update flow. 

Also it would be good to do small tutorial on how to do this, since i haven't seen some of discord using this feature.

**Comment by damskyftw** [comment]
For now i archived these 3 channels, deleted 2 completely empty ( only 2 messages in them ) and looking to reduce it more.

![Screenshot_1](https://user-images.githubusercontent.com/92786519/151573090-c1a51d63-712a-49d1-a231-0c9d138e2d09.png)

**Comment by damskyftw** [comment]
I'm definitely thinking to shrink even more channels into one - :) we are fighting for attention and with so many channels even now i feel like its too many

**Comment by damskyftw** [comment]
current state:
![11](https://user-images.githubusercontent.com/92786519/162631010-78a46bd0-e0c1-4daa-824e-57387af7ba2f.png)
![22](https://user-images.githubusercontent.com/92786519/162631011-88becc2a-7c78-4840-8619-2ebafeb07fdc.png)
